server/src/socobouser/handlers/SocoboUser.handler.ts

```diff
@@ -13,30 +13,30 @@ export class UserHandler {
   }
 
   public getAll = (req: Request, res: Response): void => {
-    this._db.users.getAll()
+    this._db.socobousers.getAll()
       .then((result: SocoboUser[]) => res.status(200).json(result))
       .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
   }
 
   public getById = (req: Request, res: Response): void => {
     const userId: number = +req.params.id;
-    this._db.users.getUserById(userId)
+    this._db.socobousers.getUserById(userId)
       .then((result: SocoboUser) => res.status(200).json(result))
       .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
   }
 
   public updateById = (req: Request, res: Response): void => {
     const userId: number = +req.params.id;
-    const updateType: UpdateType = req.body.updateType;
+    const updateType: SocoboUserUpdateTypes = req.body.updateType;
     const fieldValues: string[] = req.body.fieldValues;
-    this._db.users.updateById(userId, updateType, fieldValues)
+    this._db.socobousers.updateById(userId, updateType, fieldValues)
       .then((result: SocoboUser) => res.status(200).json(result))
       .catch((e: any) => res.status(e.statusCode).json(e.forResponse()));
   }
 
   public deleteById = (req: Request, res: Response): void => {
```

Comment:
I think for update and delete we need to check if the id of the current user is the same as the id of the user who should be updated because else it would be possible to change other users (if I know the id), or I'm, wrong?